modin/pandas/test/dataframe/test_default.py

```diff
@@ -1186,12 +1219,13 @@ def test_setattr_axes():
             # In BaseOnPython, setting columns raises a warning because get_axis
             #  defaults to pandas.
             warnings.simplefilter("error")
-        df.index = ["foo", "bar"]
-        df.columns = [9, 10]
+        if StorageFormat.get() != "Hdk":  # Not yet supported
```

Comment:
Do we have an issue for this?

Reply:
Yes - https://github.com/modin-project/modin/issues/1766. I'll add a comment.